src/Classes/RichTextViewController.swift

```diff
@@ -496,12 +496,10 @@ public class RichTextViewController: UIViewController {
     }
     
     private func removeFormattingFromListLeadsInRange(range: NSRange) {
-        guard let listHeadRegex = try? NSRegularExpression(pattern: "^(([0-9]+\\.\\u00A0)|(\\u2022\\u00A0)).*$", options: .AnchorsMatchLines),
-            regularFont = regularFont where
-            range.length > 0
-            else {
-                print("Failed to remove formatting")
-                return
+        guard let regularFont = regularFont else { return }
+        guard let listHeadRegex = try? NSRegularExpression(pattern: "^(([0-9]+\\.\\u00A0)|(\\u2022\\u00A0)).*$", options: .AnchorsMatchLines) where range.length > 0 else {
```

Comment:
Not super important, but if `range.length` is empty then there's no need to create the regex, so it would be better to do
``` swift
guard range.length > 0, let listHeadRegex = try? NSRegularExpression(pattern: "^(([0-9]+\\.\\u00A0)|(\\u2022\\u00A0)).*$", options: .AnchorsMatchLines) else {
```